build.psm1

```diff
@@ -2765,13 +2775,20 @@ function Start-CrossGen {
     # the NI PS assemblies, we need to replace the IL PS assemblies with the corresponding NI
     # PS assemblies, but with the same IL assembly names.
     #
-    Write-Verbose "PowerShell Ngen assemblies have been generated. Deploying ..." -Verbose
-    foreach ($assemblyName in $psCoreAssemblyList) {
+    Write-Verbose "PowerShell + Add-Type Ngen assemblies have been generated. Deploying ..." -Verbose
```

Comment:
I guess the purpose here is to always write out the verbose message so it's more informative to the user.

However, `PowerShell + Add-Type Ngen assemblies` looks confusing to me. I suggest we continue to use `PowerShell Ngen assemblies`.